torch/autograd/forward_ad.py

```diff
@@ -181,3 +182,18 @@ def __enter__(self):
 
     def __exit__(self, exc_type: Any, exc_value: Any, traceback: Any) -> None:
         exit_dual_level()
+
+# Private helper functions
+_set_fwd_grad_enabled = torch._C._set_fwd_grad_enabled
+_is_fwd_grad_enabled = torch._C._is_fwd_grad_enabled
+
+# Private helper function to enable or disable fwd grad.
+# If you're a user and want to use this, please file an issue to discuss the use case.
+@contextlib.contextmanager
+def _enable_fwd_grad(enabled=True):
```

Comment:
Maybe we should match the API of the current `_set_grad_enabled`, i.e. name it `_set_fwd_grad_enabled` and don't have a default value. (The `enable_grad` API currently doesn't take any arguments and always sets it to True.)

Reply:
Sure